README.md

```diff
@@ -1,3 +1,130 @@
 # You need a fahrschein to use the (nakadi event) bus
```

Comment:
What about?

```
# fahrschein

> You need a fahrschein to use the [nakadi](http://link-to-nakadi.do) event bus
```

And do not forget the [banner](http://img.theepochtimes.com/n3/eet-content/uploads/2015/03/07/20150327-steve-macchio-train-conductor-petrsvab-9003-676x450.jpg).